### ISSUE TITLE
Use Template Repository

### ISSUE THREAD

**Issue by naveen521kk** [saadpasta/developerFolio#226]
## Summary
@saadpasta Can we make this repo a [Template Repository?](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-template-repository)
This would be useful to users, who don't contribute to this repo, can create a new repo instead of forking the repo. 

### Refs
1. https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-template-repository
2. https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template

**Comment by kartikcho** [comment]
I strongly agree with this! Great idea @naveen521kk. We would need to plan a few things though, like reaching a 'version 1' of our portfolio right now and then planning features to be released in future versions etc.

**Comment by saadpasta** [comment]
@kartikcho @naveen521kk It a great idea. But I have a lot of future plans for DeveloperFolio and I don't want to make this a template right now. 

@kartikcho I agree with you we need to strongly work on our versioning and releasing

**Comment by kartikcho** [comment]
> I have a lot of future plans for DeveloperFolio and I don't want to make this a template right now.

@saadpasta it would be great if you could write a roadmap (rough or final) so we can decide where to direct development from here.

**Comment by saadpasta** [comment]
> > I have a lot of future plans for DeveloperFolio and I don't want to make this a template right now.
> 
> @saadpasta it would be great if you could write a roadmap (rough or final) so we can decide where to direct development from here.

I'll do that

**Comment by kartikcho** [comment]
Any thoughts on making this happen now. I think we've reached a good enough 'version 1' of our app. 
Maybe after #333 gets in @naveen521kk @saadpasta ?

**Comment by naveen521kk** [comment]
Seems like a good idea :)

**Comment by kartikcho** [comment]
Don't give me credit, it was your suggestion in the first place 😄
